test/Confluent.Kafka.IntegrationTests/Tests/Producer_ProduceAsync_DeliveryHandler.cs

```diff
@@ -63,6 +63,15 @@ public void HandleDeliveryReport(Message dr)
                     Assert.Equal(new byte[] { 7 }, dr.Value);
                 }
 
+                if (Count == 6)
+                {
+                    Assert.True(Math.Abs((Now - dr.Timestamp.DateTime).TotalMilliseconds) < 1.0);
```

Comment:
I didn't check this 1.0 : is it [-0.5:0.5] or [-1.0;1.0] or [0:1.0]? Not sure we care but should be in doc (especially the fact it's not the same datetime if datetime is more precise than ms)

Also need a test with local datetime